webpack.config.js

```diff
@@ -75,10 +77,60 @@ const externals = {
 	};
 } );
 
+/**
+ * Webpack plugin for handling specific template tags in Webpack configuration
+ * values like those supported in the base Webpack functionality (e.g. `name`).
+ *
+ * @see webpack.TemplatedPathPlugin
+ */
+class CustomTemplatedPathPlugin {
+	/**
+	 * CustomTemplatedPathPlugin constructor. Initializes handlers as a tuple
+	 * set of RegExp, handler, where the regular expression is used in matching
+	 * a Webpack asset path.
+	 *
+	 * @param {Object.<string,Function>} handlers Object keyed by tag to match,
+	 *                                            with function value returning
+	 *                                            replacement string.
+	 *
+	 * @return {void}
+	 */
+	constructor( handlers ) {
+		this.handlers = reduce( handlers, ( result, handler, key ) => {
+			const regexp = new RegExp( `\\[${ escapeRegExp( key ) }\\]`, 'gi' );
+			return [ ...result, [ regexp, handler ] ];
+		}, [] );
+	}
+
+	/**
+	 * Webpack plugin application logic.
+	 *
+	 * @param {Object} compiler Webpack compiler
+	 *
+	 * @return {void}
```

Comment:
Do we really need to explicitly document that function doesn't return anything? Wouldn't it enough to assume that a function by default doesn't return value?

Reply:
>Do we really need to explicitly document that function doesn't return anything? Wouldn't it enough to assume that a function by default doesn't return value?

There's some precedent of this in existing core documentation. In general I'd be inclined to agree with you that it could be omitted for `undefined` return, but I could also see an argument for consistency / explicitness. Maybe worth bringing up in tomorrow's JS meeting? 😉

Reply:
Docs again? Count me in ❤️